tests/st/codegen/dsl/test_ptoas_multi_buffer_codegen.py

```
@@ -0,0 +1,189 @@
+    cfg = RunConfig(
+        platform="a2a3",
+        codegen_only=True,
+        save_kernels=True,
+        save_kernels_dir=str(dump_dir),
+    )
+    ctx = passes.PassContext([], use_ptoas_multi_buffer=use_multi_buffer)
+    with ctx:
+        try:
+            kernel(torch.randn(T, N), torch.empty(T, N), config=cfg)
+        except Exception:  # noqa: BLE001 - post-codegen (simpler) failures are irrelevant here
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)